app/components/save_button_component.rb

```diff
@@ -0,0 +1,8 @@
+class SaveButtonComponent < ApplicationComponent
```

Comment:
How about `SubmitButtonComponent`?

Reply:
I like it, it covers more cases than just saving a record.